tasks.c

```diff
@@ -1075,6 +1077,7 @@ static void prvYieldForTask( TCB_t * pxTCB,
 
         if( ( pxTaskBuffer != NULL ) && ( puxStackBuffer != NULL ) )
         {
+            prvTaskCreator( pxTaskCode, ulStack)
```

Comment:
Is this function / macro defined anywhere?

Reply:
Thanks.  This one is critical.